src/components/dashboard/dashboard.module.css

```diff
@@ -10,6 +10,13 @@
   z-index: 999;
 }
 
+.branding {
+  content:url('assets/branding.svg');
+  position: absolute !important;;
```

Comment:
there's an extra semicolon here

Reply:
Also, do we have to use !important here? why is that?

Reply:
cause header button had important lol, but I don't think it's necessary